OpenRA.Game/Map/MapPreview.cs

```diff
@@ -335,7 +355,7 @@ public MapPreview(Map map, ModData modData)
 			innerData.SetCustomRules(modData, this, new Dictionary<string, MiniYaml>()
 			{
 				{ "Rules", map.RuleDefinitions },
-				{ "Translations", map.TranslationDefinitions },
+				{ "FluentMessages", map.FluentMessageDefinitions },
```

Comment:
This is also worse. We don't call it `MiniYamlRules` but just `Rules`. Please keep this as is for consistency.

Reply:
It is definitely confusing. I almost feel like just having it be `Fluent` would be better

Reply:
https://github.com/OpenRA/OpenRA/pull/21622#pullrequestreview-2375136239

Reply:
> In Fluent, the basic unit of translation is called a message. Messages are containers for information. You use messages to identify, store, and recall translation information to be used in the product.

This node is listing filenames containing/inline definitions of fluent messages. This is the same as "Sequences" listing filenames containing/inline definitions of sequences, etc.

There is no logic here to handle multiple languages, which I feel is a basic requirement for calling something "Translations".

Reply:
The definition already says that it is `TranslationMessages`. People who don't know that @projectfluent is a translation framework will be irritated. Ideally you should revert the renamings from #21602 as well.

Reply:
That would conflict with the naming of things in the actual translation system that manages the sets of language-specific fluent / sound / sprite files.

Reply:
On a yaml level I think this is the only place it is called fluent? So for non-power users both the concept of fluent and messages might be foreign

Reply:
for more advanced modders the other place will be https://github.com/OpenRA/OpenRA/pull/21622#discussion_r1807804555. I think we should make sure these are named well.